gcs/CHANGES.md

```diff
@@ -21,6 +21,10 @@
 
 1. Increasing gRPC read timeout from 30 seconds to 3600 seconds.
 
+1. Updating max list items per call from 1024 -> 5000. (#872)
+
+1. Adding support for invocationId in Json path. (#873)
```

Comment:
Remove the PR number from the description.

Reply:
addressed.